app/views/contexts/summary_context.py

```diff
@@ -111,7 +114,29 @@ def build_view_context_for_section_summary(
         [section],
     )
 
-    context['summary'].update({'title': title})
+    list_collector_blocks = schema.get_visible_list_blocks_for_section(section)
+
+    list_summaries = []
+
+    for list_collector_block in list_collector_blocks:
+        list_summary = {
+            'title': list_collector_block['summary']['title'],
```

Comment:
We need to add support to placeholder rendering for `title` as on census it pipes in the census date.